downstream/modules/platform/ref-access-control-requirements.adoc

```diff
@@ -0,0 +1,176 @@
+
+[id="ref-access-control-requirements_{context}"]
+
+= Access control requirements
+
+[role="_abstract"]
+
+{PlatformName} uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by a firewall.
+
+The following tables provide the default {PlatformName} destination ports required for each application.
+
+.PostgreSQL
+[options="header"]
+|===
+|Port |Protocol |Service |Required for
+|22
+|TCP
+|SSH
+|Remote access during installation
+|5432
+|TCP
+|Postgres
+a|Default port
+
+ALLOW connections from controller(s) to database port
+|===
+
+.{ControllerNameStart}
+[options="header"]
+|===
+|Port |Protocol |Service |Required for
+|22
+|TCP
+|SSH
+|Installation
+
+|80
+|TCP
+|HTTP
+|UI/API & {ExecEnvShort} pulls
+
+|443
+|TCP
+|HTTPS
+|UI/API & {ExecEnvShort} pulls
+
+|5432
+|TCP
+|PostgreSQL
+a|Open *only* if the internal database is being used. Otherwise, this port should not be opened
+
+Hybrid mode in a cluster
+
+|27199
+|TCP
+|Receptor
+|ALLOW receptor port across all controllers (for mandatory & automatic control plane peering)
+|===
+
+.Hop Nodes
+[options="header"]
+|===
+|Port |Protocol |Service |Required for
+|22
+|TCP
+|SSH
+|Installation
+
+|27199
+|TCP
+|Receptor
+a|Mesh
+
+ALLOW connection from controller(s) to receptor port
+|===
+
+.Execution Nodes
+[options="header"]
+|===
+|Port |Protocol |Service |Required for
+|22
+|TCP
+|SSH
+|Installation
+
+|80
+|TCP
+|HTTP
+|Execution Environment (EE) pulls container images from registry
+
+|443
```

Comment:
ditto

Reply:
@Spredzy Thanks, I'll remove ports 80 and 443 from Execution Nodes.